components/gems/Gemfile.lock

```diff
@@ -292,7 +292,7 @@ GEM
       tty-spinner
     chef-bin (15.3.14)
       chef (= 15.3.14)
-    chef-cli (1.0.13)
+    chef-cli (1.0.14)
```

Comment:
🚧 This isn't ready to be pulled in (by my understanding) until https://github.com/chef/chef-workstation/pull/479 is merged